contracts/test/TestERC20.sol

```diff
@@ -1,80 +1,32 @@
 pragma solidity ^0.5.4;
 
-/**
- * ERC20 contract interface.
- */
-contract ERC20 {
-    function totalSupply() public view returns (uint);
-    function balanceOf(address tokenOwner) public view returns (uint balance);
-    function allowance(address tokenOwner, address spender) public view returns (uint remaining);
-    function transfer(address to, uint tokens) public returns (bool success);
-    function approve(address spender, uint tokens) public returns (bool success);
-    function transferFrom(address from, address to, uint tokens) public returns (bool success);
-
-    event Transfer(address indexed from, address indexed to, uint tokens);
-    event Approval(address indexed tokenOwner, address indexed spender, uint tokens);
-}
+import "openzeppelin-solidity/contracts/token/ERC20/ERC20.sol";
 
 /**
  * ERC20 test contract.
  */
 contract TestERC20 is ERC20 {
 
-    string constant public symbol = "AGT";
-    string constant public name = "ArgentToken";
-    uint8 public decimals;
-    uint internal _totalSupply;
-
-    mapping(address => uint) balances;
-    mapping(address => mapping(address => uint)) allowed;
-
-    constructor(address[] memory _owners, uint _supply, uint8 _decimals) public {
-        decimals = _decimals;
-        for(uint i = 0; i < _owners.length; i++) {
-            balances[_owners[i]] = _supply * 10**uint(_decimals);
+    constructor (address[] memory _initialAccounts, uint _supply, uint8 _decimals) public {
+        for(uint i = 0; i < _initialAccounts.length; i++) {
+            _mint(_initialAccounts[i], _supply * 10**uint(_decimals));
         }
-        _totalSupply = _owners.length * _supply * 10**uint(_decimals);
     }
 
-    function totalSupply() public view returns (uint) {
-        return _totalSupply  - balances[address(0)];
+    function mint(address account, uint256 amount) public {
+        _mint(account, amount);
     }
 
-    function balanceOf(address tokenOwner) public view returns (uint balance) {
-        return balances[tokenOwner];
+    function burn(address account, uint256 amount) public {
+        _burn(account, amount);
     }
 
-    function transfer(address to, uint tokens) public returns (bool success) {
-        require(balances[msg.sender] >= tokens);
-        balances[msg.sender] = balances[msg.sender] - tokens;
-        balances[to] = balances[to] + tokens;
-        require(balances[to] + tokens > balances[to]);
-        emit Transfer(msg.sender, to, tokens);
-        return true;
+    function burnFrom(address account, uint256 amount) public {
+        _burnFrom(account, amount);
     }
 
-    function approve(address spender, uint tokens) public returns (bool success) {
-        allowed[msg.sender][spender] = tokens;
-        emit Approval(msg.sender, spender, tokens);
-        return true;
+    function approveInternal(address owner, address spender, uint256 value) public {
```

Comment:
why do we need that?

Reply:
To be honest I just copied their mock contract https://github.com/OpenZeppelin/openzeppelin-solidity/blob/master/contracts/mocks/ERC20Mock.sol without giving it much thought. I'll remove it.